conf/container.config

```diff
@@ -0,0 +1,18 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Nextflow base config file
+ * -------------------------------------------------
+ * A 'blank slate' config file, appropriate for general
+ * use on most high performace compute environments.
+ * Assumes that all software is installed and available
+ * on the PATH. Runs in `local` mode - all jobs will be
+ * run on the logged in environment.
+ */
+
+
+
+docker {
```

Comment:
Can we just add this one line in the main `nextflow.config` instead of having its own file? (`docker.registry = `)